src/js/views/patients/shared/components/dialer_component.js

```diff
@@ -0,0 +1,101 @@
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+import hbs from 'handlebars-inline-precompile';
+import { Component } from 'marionette.toolkit';
+
+import intl from 'js/i18n';
+
+import 'scss/modules/buttons.scss';
+
+import Optionlist from 'js/components/optionlist';
+
+import './dialer-component.scss';
+
+const i18n = intl.patients.shared.components.dialerComponent;
+
+export default Component.extend({
+  initialize({ action }) {
+    this.field = this.getField(action);
+  },
+  getField(action) {
+    const patient = action.getPatient();
+
+    return Radio.request('entities', 'patientFields:model', {
+      name: 'phones',
+      _patient: patient.getResource(),
+    });
+  },
+  getLists() {
+    if (this._lists) return this._lists;
+
+    this._lists = new Promise(resolve => {
+      this.field.fetch().then(() => {
+        const phones = this.getPhones();
+        resolve([{ collection: phones }]);
+      });
+    });
+
+    return this._lists;
+  },
+  getPhones() {
+    const phones = new Backbone.Collection(this.field.get('value'), {
+      comparator(model) {
+        return model.get('preferred') ? 0 : 1;
+      },
+    });
+
+    phones.each(model => {
+      model.set('onSelect', () => {
+        Radio.request('dialer', 'call', model.get('number'));
```

Comment:
ah one other thing the 4th argument for this request needs to be the action model @nmajor25